vmdb/app/models/miq_expression.rb

```diff
@@ -1514,24 +1515,26 @@ def self.build_relats(model, parent={}, seen=[])
     result
   end
 
-  def self.get_table_details(table)
-# puts "Enter: get_table_details: model: #{model}, parent: #{parent.inspect}"
-    [self.value2human(table), table]
+  def self.get_table_details(table_path, table_assoc = nil)
+    table_assoc = table_assoc.nil? ? table_path : table_path.gsub(/(.*)(\.)(.*)/,"\\1\\2#{table_assoc.to_s}")
```

Comment:
This line is duplicated below in get_column_details, so thinking this should move into a method with a nice name....what that name is I have no idea :stuck_out_tongue: